make.js

```diff
@@ -1459,7 +1459,15 @@ target.lint = function() {
   var jshintPath = path.normalize('./node_modules/.bin/jshint');
   if (!test('-f', jshintPath)) {
     echo('jshint is not installed -- installing...');
-    exec('npm install jshint@2.4.x'); // TODO read version from package.json
+    // Read the jshint version to be installed from package.json.
+    fs.readFile('package.json', 'utf8', function (error, data) {
```

Comment:
Done. I think this also makes the code a bit cleaner and indeed it fixes the problem that one would have to re-run `node make lint` when it was not yet installed. I have tested this with `jshint` installed, with `jshint` uninstalled and with `jshint` uninstalled and no `package.json` and all possibilities yield an expected result now. Excellent, thank you!